Cargo.toml

```diff
@@ -11,3 +11,5 @@ authors = ["Artem Zinnatullin <artem.zinnatullin@gmail.com>", "Artur Dryomov <ar
 [dependencies]
 yaml-rust = "0.4.2"
 linked-hash-map = "0.5.1"
+crossbeam-channel = "0.3"
```

Comment:
I haven't used it in a while, but any reason not to use `std::sync::mpsc`?

Reply:
Initially I swapped to `crossbeam` because I needed multi-receiver support, but then even that wasn't enough so I've also added `bus`

Decided to keep crossbeam for performance lol (nanoseconds matter hey!), we'll be stripping final binary #242 so it shouldn't matter which one we're using from the binary size point